README.md

```diff
@@ -1,6 +1,6 @@
 # TITAN Simulation
 
-![](https://github.com/marshall-lab/TITAN/workflows/Unit%20Tests/badge.svg) [![codecov](https://codecov.io/gh/marshall-lab/TITAN/branch/develop/graph/badge.svg?token=wjkExshhyh)](https://codecov.io/gh/marshall-lab/TITAN)
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/marshall-lab/TITAN) ![](https://github.com/marshall-lab/TITAN/workflows/Unit%20Tests/badge.svg) [![codecov](https://codecov.io/gh/marshall-lab/TITAN/branch/develop/graph/badge.svg?token=wjkExshhyh)](https://codecov.io/gh/marshall-lab/TITAN) ![GitHub](https://img.shields.io/github/license/marshall-lab/TITAN) [![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://https://marshall-lab.github.io/TITAN/)
```

Comment:
docs link is broken.  It would be nice if clicking on the release badge took you to the release page, the license page the license.md file, and the tests the actions tab